tests/TCP_echo_client.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, prefix, host, port, size, count,
                  timeout=TIMEOUT,
                  logger=None,
                  ssl_info=None,
-                 delay_close=False):
+                 delay_close=False,
+                 wait_connected=True):
```

Comment:
Just curious as to what is the use case when an echo client will be instantiated with wait_connected=False ?

Reply:
That's the original behavior - it never waited for the client to connect. Left that possible in case any of the existing tests broke but none have.  I could simply remove it and make the default wait. In the future if any test was written that expected the client to not connect (like test removing a listener) then we'd need to add it back.

Reply:
> I could simply remove it and make the default wait. In the future if any test was written that expected the client to not connect (like test removing a listener) then we'd need to add it back.

Yes. that is most excellent. Let's remove the flag and make the default wait.